tests/foundry/exclude

```diff
@@ -4,6 +4,7 @@ AccountParamsTest.testFail_GetNonce_false
 AddrTest.test_addr_false
 AddrTest.testFail_addr_false
 AddrTest.testFail_addr_true
+AddrTest.test_addr_symbolic
```

Comment:
Let's also either add a wrapper rule around the `addr` cheatcode with the  `[concrete]` tag, in a similar way to how `#sign` is defined and used. Either in this PR or in a new one. Just left this comment to keep track of the issue.

Reply:
Actually I already tried that, but Everett suggested to put the `concrete` tag on `#addrFromPrivateKey` directly. But the problem for symbolic inputs remained. Let's open another issue to fix this.